tests/storage/migration.go

```diff
@@ -436,14 +445,19 @@ var _ = Describe(SIG("Volumes update with migration", decorators.RequiresTwoSche
 				return claim == destDV.Name
```

Comment:
I see a lot of checks like this in the code, I believe it will be better to change it to return the string claim, and update Should(Equal(destDV.Name)) to have a better debugable message

Reply:
Yes, I wanted to refactor it in a separate PR, but I addressed them in the last commit